### DISCUSSION TITLE
Avast antivirus blocks site hosted on Vercel

### DISCUSSION THREAD

**Discussion by rmnmsr** [vercel/community#5244] [Help]
### Summary

Hi,

We recently launched [our site](www.flipflopesports.com/en) through Vercel and for a while the site could be accessed without problems.

Recently we get complains from clients who use Avast antivirus that they cannot access the website because Avast evaluates it as unsecure.

My thoughts on this are that it could be something with SSL certificates but I am no expert on this and require help.

### Steps to Reproduce

- Download Avast antivirus
- Access the website www.flipflopesports.com/en
- (As you type in the site url Avast already detects it as unsecure... you dont even have to press enter)

**Comment by amyegan** [comment]
Hi @rmnmsr. I can't be sure of the exact reason in this case since I don't have access to their systems to investigate the block. But I found a similar question in their forum with some suggestions at [forum.avast.com/index.php?topic=199595.0](https://forum.avast.com/index.php?topic=199595.0).

You can report a false positive at [avast.com/false-positive-file-form.php](https://www.avast.com/false-positive-file-form.php). And you can contact the Avast Support team at [support.avast.com/en-us/index](https://support.avast.com/en-us/index). They should be able to help you identify why their product is blocking your site. 🙂
